playbooks/roles/airship-setup-deployer/tasks/main.yml

```diff
@@ -17,6 +17,17 @@
   tags:
     - install
 
+- name: Install client packages
+  package:
+    name: "{{ suse_osh_client_packages }}"
+    state: present
+  register: install_client_packages
```

Comment:
is that var used anywher?

Reply:
it is not. I think this is there so that lint does not complain